worldedit-core/src/main/java/com/sk89q/jnbt/ByteArrayTag.java

```diff
@@ -61,4 +56,11 @@ public String toString() {
         return "TAG_Byte_Array(" + hex + ")";
     }
 
+    // FAWE Start
+    @Override
```

Comment:
Why create a diff here just to move where in the file it is? Like I agree the place it's at in the WE-bundled JNBT is bad, but seems like a pretty pointless diff to create here, especially when JNBT is likely being dropped in WE8